services/galley/src/Galley/Intra/BackendNotificationQueue.hs

```diff
@@ -1,6 +1,6 @@
 {-# LANGUAGE NumericUnderscores #-}
 
-module Galley.Intra.BackendNotificationQueue where
+module Galley.Intra.BackendNotificationQueue (interpretBackendNotificationQueueAccess) where
```

Comment:
why?

Reply:
Just to prevent code completion from suggesting the wrong function.